packages/babel-core/test/api.js

```diff
@@ -146,7 +146,7 @@ describe("api", function () {
           plugins: [__dirname + "/../../babel-plugin-syntax-jsx", false],
         });
       },
-      /TypeError: \[BABEL\] unknown: Falsy value found in plugins/
```

Comment:
nvm I was talking about it saying it was a plugin/preset before but maybe that's not a big deal